CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -21,20 +21,16 @@ public static class EnvUtil
         public const string WindowsIntegratedAuthenticationEnvVar = "NUGET_CREDENTIALPROVIDER_WINDOWSINTEGRATEDAUTHENTICATION_ENABLED";
         public const string ForceCanShowDialogEnvVar = "NUGET_CREDENTIALPROVIDER_FORCE_CANSHOWDIALOG_TO";
 
-        public const string AuthorityEnvVar = "NUGET_CREDENTIALPROVIDER_ADAL_AUTHORITY";
-        public const string AdalFileCacheEnvVar = "NUGET_CREDENTIALPROVIDER_ADAL_FILECACHE_ENABLED";
-        public const string PpeHostsEnvVar = "NUGET_CREDENTIALPROVIDER_ADAL_PPEHOSTS";
```

Comment:
>NUGET_CREDENTIALPROVIDER_ADAL_PPEHOSTS [](http://example.com/codeflow?start=46&length=38)

This one had nothing to do with ADAL, so renaming it. Technically a back-compat change, but considering it's for PPE the existing usage should be low